### ISSUE TITLE
Unable to change font size when Dyslexia friendly font is enabled

### ISSUE THREAD

**Issue by RoseReatherford** [openlibhums/janeway#4684]
**Describe the bug**
When enabling Dyslexia friendly fonts, changing the font size does not affect the body of a text.

**Janeway version**
1.7.7

**To Reproduce**
1. Go to any article
2. Click on "Dyslexia"
3. Click on "A+" or "A-"
4. See the font size is not changed in the body of the text.

**Expected behavior**
1. Go to any article
2. Click on "Dyslexia"
3. Click on "A+" or "A-"
4. See the font size is changed in the body of the text.

## Front-end Issues

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser Chrome
 - Version 134.0.6998.165
